documentation/developers.md

```diff
@@ -78,6 +78,7 @@ webserver: {
     ssl-key: /etc/ssl/private_keys/myhostname.pem
     ssl-ca-cert: /etc/ssl/certs/ca.pem
     default-server: true
+    client-auth: "want"
```

Comment:
@justinstoller is this needed?

Reply:
This was required for your test that created an https request but didn't provide it's own cert. (we wanted to verify it got a 403 rather than https error). I don't believe it's required for this documentation.